src/pages/registration/contributors_tab/components/AuthorList.tsx

```diff
@@ -21,38 +21,34 @@ const AuthorTable: FC<AuthorTableProps> = ({ authors, onDelete, onMoveAuthor, op
   };
 
   return (
-    <TableContainer>
-      <Table>
-        <TableBody>
-          {authors.map((author: Contributor) => (
-            <AuthorCard
-              author={author}
-              key={author.identity.id || author.identity.name}
-              onMoveAuthor={(event) => onMoveAuthor(event.target.value - 1, author.sequence - 1)}
-              onRemoveAuthorClick={() => setAuthorToRemove(author)}
-              openContributorModal={openContributorModal}
-            />
-          ))}
-        </TableBody>
-      </Table>
+    <>
+      {authors.map((author: Contributor) => (
```

Comment:
vil tro authors allerede har type?
```suggestion
      {authors.map((author) => (
```